src/pages/Popup/Popup.tsx

```diff
@@ -29,7 +35,7 @@ const Popup = () => {
     }
     setTabs(filteredTabs);
     return filteredTabs;
-  }, [repo]);
+  }, [repo, uniqueUrls]);
```

Comment:
suggestion: with the above suggested solution, we could revert this line back (removing `uniqueUrls`)

Reply:
Committed your suggested changes.